packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -686,6 +686,15 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     );
   }
 
+  /// This method is called by [BallisticScrollActivity.applyNewDimensions].
+  /// It doesn't affect NestedScrollView. So this method uses an empty implementation.
+  /// Maybe call this method in [_NestedOuterBallisticScrollActivity] and [_NestedInnerBallisticScrollActivity]
```

Comment:
These public docs (`///`) won't be helpful since these are private classes.

`//` are private

```suggestion
  // Unsupported
```